products/compute/examples/chef/interconnect_attachment.rb

```diff
@@ -0,0 +1,41 @@
+<% if false # the license inside this if block assertains to this file -%>
+# Copyright 2017 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+<% end -%>
+<% unless name == 'README.md' -%>
+
+<%= compile 'templates/license.erb' -%>
+
+<%= lines(autogen_notice :chef) -%>
+
+<%= compile 'templates/chef/example~auth.rb.erb' -%>
+
+gcompute_region 'us-central1' do
+ project 'google.com:graphite-playground'
+ credential 'mycred'
+end
+
+<% else -%>
+# Tip: Be sure to include a valid gcompute_disk object
+<% end # name == README.md -%>
+gcompute_interconnect_attachment <%= example_resource_name('test-interconnect') -%> do
+  action       :create
+  project      'google.com:graphite-playground'
+  region       'us-central1'
+  name         '<%= example_resource_name('test-interconnect') -%>'
+  interconnect 'https://googleapis.com/compute/v1/projects/...global/interconnects/...'
+  router       'https://googleapis.com/compute/v1/projects/...regions/.../routers/...'
```

Comment:
What are the ...s about?

Reply:
I can use `$variables` instead - the idea is that the user has to set those.